mithril-test-lab/mithril-devnet/devnet-mkfiles.sh

```diff
@@ -584,20 +587,31 @@ CARDANO_NODE_SOCKET_PATH=node-pool${N}/ipc/node.sock ./cardano-cli transaction s
     --tx-file node-pool${N}/tx/tx${N}.tx \\
     --testnet-magic ${NETWORK_MAGIC}
 
+EOF
+
+  # Copy retrieve transaction id to activate.sh script
```

Comment:
Is that normal that it overrider the `CARDANO_NODE_SOCKET_PATH` retrieved with the previous command ?